<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogForm</class>
 <widget class="QMainWindow" name="LogForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monaco</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Reginas | For Employees</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>241</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>36</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Reginas</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="buddy">
     <cstring>label</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For Employees&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLog">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 49, 113);
color: white;
border-radius: 10px;

</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReg">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 49, 113);
color: white;
border-radius: 10px;

</string>
    </property>
    <property name="text">
     <string>REGISTER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPas">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;
border-radius: 10px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLog">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;
border-radius: 10px;

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLogReg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;
border-radius: 10px;

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPasReg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;
border-radius: 10px;

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKeyReg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;
border-radius: 10px;

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>300</y>
      <width>600</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 49, 113);
border-radius: 10px;

</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditLog</tabstop>
  <tabstop>lineEditPas</tabstop>
  <tabstop>pushButtonLog</tabstop>
  <tabstop>lineEditLogReg</tabstop>
  <tabstop>lineEditPasReg</tabstop>
  <tabstop>lineEditKeyReg</tabstop>
  <tabstop>pushButtonReg</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
